UPGRADE.md

```diff
@@ -81,3 +81,4 @@ references:
 ## Origin 1.1.x / OSE 3.1.x
 
 1. The `buildconfig` label on Build objects, which contains the name of the BuildConfig for the Build, has been deprecated in favor of a new `openshift.io/build-config.name` label.
+2. New Service Accounts specific to the Persistent Volume operations of binding, recycling, and provisioning were added.  Run `oadm policy reconcile-sccs --confirm` to update your Security Contexts and generate the new Service Accounts.
```

Comment:
'service accounts'
'PersistentVolume'
'SecurityContextConstraints'
this doesn't generate the new service accounts